web/src/app/administration/classification/details/classification-details.component.ts

```diff
@@ -92,6 +92,7 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
     })
     this.classificationSelectedSubscription = this.classificationsService.getSelectedClassification()
       .subscribe(classificationSelected => {
+        if (this.classification ? this.classification.classificationId === classificationSelected.classificationId : false) { return; }
```

Comment:
Do not use ? : operator withing an if sentence to break the flow, keep it simple, just use short evaluation like this.
if (this.classification && this.classification.classificationId === classificationSelected.classificationId ) { return; }